### PR TITLE
[EmbeddingApi-usecase] Add usecase for UIClient.openFileChooser to op…

### PR DESCRIPTION
**Pull request by jacky-young**
…en local file

-Add usecase for UIClient.openFileChooser to open local file
-Cover the XWalkActivity and XWalkInitializer two ways

Impacted tests(approved): new 2, update 0, delete 0
Unit test platform: [Android]
Unit test result summary: pass 2, fail 0, block 0

BUG=https://crosswalk-project.org/jira/browse/XWALK-5212

### CODE REVIEW

**Comment by jacky-young** [comment]
@jennycao @Shao-Feng please help to review.

File: usecase/usecase-embedding-android-tests/embeddingapi/src/org/xwalk/embedded/api/sample/XWalkViewWithOpenFileChooser.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        mImageView = (ImageView) findViewById(R.id.imageview);
+    }
+
+    @Override
+    protected void onXWalkReady() {
+        StringBuffer mess = new StringBuffer();
+        mess.append("Test Purpose: \n\n")
+        .append("Verifies XWalkView can open local file.\n\n")
+        .append("Expected Result:\n\n")
+        .append("Test passes if app load picture from your storage gallery.");
```
Comment (Shao-Feng) [inline usecase/usecase-embedding-android-tests/embeddingapi/src/org/xwalk/embedded/api/sample/XWalkViewWithOpenFileChooser.java]: Need more steps for choosing a picture here

File: usecase/usecase-embedding-android-tests/embeddingapi-async/src/org/xwalk/embedded/api/asyncsample/XWalkViewWithOpenFileChooserAsync.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        public void openFileChooser(XWalkView view, ValueCallback<Uri> uploadFile,
+                String acceptType, String capture) {
+            super.openFileChooser(view, uploadFile, acceptType, capture);
+            mUploadMessage = uploadFile;
+        }
+    }
+
+
+    @Override
+    protected void onActivityResult(int requestCode, int resultCode, Intent intent) {
```
Comment (Shao-Feng) [inline usecase/usecase-embedding-android-tests/embeddingapi-async/src/org/xwalk/embedded/api/asyncsample/XWalkViewWithOpenFileChooserAsync.java]: I don't understand why this method is overrided. if you just hope to load and show the choosed image. Why not add a handler to do it when the 'file' form is changed?
as below 

<input type="file" id="fileInput" />


$('#fileInput').on('change', function() {
    alert(this.files)  // or alert(this.files[0]) 
})

Comment (jacky-young) [inline usecase/usecase-embedding-android-tests/embeddingapi-async/src/org/xwalk/embedded/api/asyncsample/XWalkViewWithOpenFileChooserAsync.java]: this case tests UIClient.openFileChooser method work right or not, from a local html with input file type. Choose a system gallery image, and show it on the view. It needs override openFileChooser and onActivityResult of course. This is the test case point.

Comment (Shao-Feng) [inline usecase/usecase-embedding-android-tests/embeddingapi-async/src/org/xwalk/embedded/api/asyncsample/XWalkViewWithOpenFileChooserAsync.java]: OK, I see. this method will be invoked once the FileChooser is closed

**Comment by jacky-young** [comment]
@Shao-Feng please review again. Thanks.

**Pull request merged**